MinecraftClient/ChatBots/AutoAttack.cs

```diff
@@ -19,6 +19,21 @@ class AutoAttack : ChatBot
         private int attackRange = 4;
         private Double serverTPS;
         private float health = 100;
+        private bool singleMode = true;
+        private bool priorityDistance = true;
+
+        public AutoAttack(string mode, string priority)
+        {
+            if (mode == "single")
+                singleMode = true;
+            else if (mode == "multi")
+                singleMode = false;
+
+            if (priority == "distance")
+                priorityDistance = true;
+            else if (priority == "health")
+                priorityDistance = false;
```

Comment:
else tell the user that `priority` is not valid? (and optionally unload the bot)